manifests/server.pp

```diff
@@ -25,6 +25,7 @@
   $database_username                      = $puppetdb::params::database_username,
   $database_password                      = $puppetdb::params::database_password,
   $database_name                          = $puppetdb::params::database_name,
+  $database_ip_mask_allow_all_users       = $puppetdb::params::database_ip_mask_allow_all_users,
```

Comment:
The puppetdb module configures two things generally, the "database" (ie. the postgresql server and database) and the "server" which is the PuppetDB webserver. So this database configuration does not need to be in the `server.pp` parameters.

I think what we need instead is for this parameter to be added the `init.pp` and then passed into the class invocation of `puppetdb::database::postgresql`, which is at the end of `init.pp`.